packages/web-components/src/components/video-player/video-player.ts

```diff
@@ -226,7 +234,9 @@ class C4DVideoPlayer extends FocusMixin(StableSelectorMixin(LitElement)) {
       ${hideCaption
```

Comment:
Just above here, on line 233, we're missing a `part` attribute. Lets add `part="video-container"` there.